### DISCUSSION TITLE
Telegramm documentation

### DISCUSSION THREAD

**Discussion by ChrisVonFrosta** [arendst/Tasmota#20034] [Q&A]
Is there any documentation for the telegram integration? i only found the comments in the file 'xdrv_40_telegram.ino'

**Comment by sfromis** [comment]
Not more than that, outside of what threads you may find searching for it.

**Reply by ChrisVonFrosta** [reply]
ok thx.
i compiled a copy of webcam with 'USE_TELEGRAM'
is it possible to send a snapshot over telegram?
